### PR TITLE
chore: put stuff back after release/8.0.2

### PR DESCRIPTION
**Pull request by pvinis**
⚠️  Merge after #7072 is merged ⚠️ 

This PR resolves []

### Description

Updates after the release (hotfixes and script updates)

### QA Test Case(s)

| Test Case | Feature | Environment | Acceptance Criteria | Setup Instructions/Link |
| --------- | :-----: | ----------: | ------------------: | ----------------------: |
|           |         |             |                     |                         |

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -147,7 +147,7 @@ GEM
     faraday_middleware (1.2.0)
       faraday (~> 1.0)
     fastimage (2.2.6)
-    fastlane (2.206.2)
+    fastlane (2.208.0)
```
Comment (pvinis) [inline Gemfile.lock:150]: we needed this for android release

File: src/app/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx [hunk 2]
```diff
@@ -159,7 +159,7 @@ export const OnboardingCreateAccountWithEmail: React.FC<OnboardingCreateAccountP
             onStateChange={(state) => {
               const routes = state?.routes
               const index = state?.index
-              if (index && routes) {
```
Comment (pvinis) [inline src/app/Scenes/Onboarding/OnboardingCreateAccount/OnboardingCreateAccount.tsx:162]: this is from #7073

**Review by MrSltun — APPROVED** [review]
Thank you Pavlos! 🌟

File: Gemfile.lock (hunk 1 above)
Comment (brainbicycle) [reply Gemfile.lock:150]: oh nice does this fix this issue?
https://artsyproduct.atlassian.net/browse/MOPLAT-330

**Review by brainbicycle — APPROVED** [review]

**Pull request merged**